docs/config-file-ort-yml.md

```diff
@@ -42,7 +42,7 @@ To be able to show why a part is excluded, each exclude must include an explanat
 The explanation consists of:
 
 * `reason` -- must be selected from a predefined list of options.
-* `comment` -- free text that provides an additional explanation.
+* `comment` -- (optional) free text that provides an additional explanation.
```

Comment:
Nit: Maybe just replace "additional" with "optional" in the sentence"? Or put "(optional)" at the end instead?

Reply:
Done.